core/domain/wipeout_service.py

```diff
@@ -39,6 +40,14 @@
 
 MAX_NUMBER_OF_OPS_IN_TRANSACTION = 25
 
+PROFILE_SPECIFIC_USER_MODELS = [
+    user_models.UserAuthDetailsModel, user_models.UserSkillMasteryModel,
+    user_models.StoryProgressModel, user_models.CollectionProgressModel,
+    user_models.LearnerPlaylistModel, user_models.ExpUserLastPlaythroughModel,
+    user_models.UserSettingsModel, user_models.CompletedActivitiesModel,
+    user_models.IncompleteActivitiesModel
+]
```

Comment:
As I think of it, could we add a new `@staticmethod` to models that will specify the minimal role that can own this model, it can be set to `EXPLORATION_EDITOR` by default in `BaseModel` and overridden for these models to be set to `ROLE_ID_LEARNER`.

Reply:
@vojtechjelinek So you suggest that instead of keeping a separate role check inside wipeout service file, we can simply see if user's role (specified in PendingDeletionRequestModel) is >= class specified role (from static method) or not to apply deletion and get away with this list? 
If this is it, then it seems good to me.

Reply:
Yep, make sense.

Reply:
Please implement the `@staticmethod` in the model classes, this list is very easy to be forgotten about when we add some new model.

Reply:
Implemented the `@staticmethod` now. PTAL at the changes!